.gitignore

```diff
@@ -3,3 +3,7 @@
 /core/target/
 /k8dns-runtime/target/
 /example/target/
+benchmarks/target/
```

Comment:
Maybe replace all those by a `**/target/`?

Reply:
Yeah the entire file can be replaced with that.